styles/docs.css

```diff
@@ -33,6 +34,12 @@ code {
 .mt-3 {
   margin-top: 3rem;
 }
+.flex-center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
```

Comment:
These can be moved into utility css as these are used regularly

Reply:
I don't have a separate utility classes CSS file, will make it soon & shift them there.